src/routes/NewUser/components/NewUserPage/NewUserPage.js

```diff
@@ -82,9 +82,11 @@ function NewUser() {
 
   if (user && user.uid) {
     defaultValues.email = user.email;
-    const name = user.displayName.split(' ');
-    [defaultValues.firstName] = name;
-    defaultValues.lastName = name.length > 1 ? name[1] : '';
+    const name = user.displayName?.split(' ');
```

Comment:
Nit pick: it might be good to call this `nameParts` or `nameArr` to make it clear what it is (name implies a string name).

I like the use of optional chaining!